src/GitHub.VisualStudio/Views/GitHubPane/FileNameToImageMonikerConverter.cs

```diff
@@ -19,6 +22,10 @@ public FileNameToImageMonikerConverter()
 
         public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
         {
+            // In design mode, imageService will be null
+            if (DesignerProperties.GetIsInDesignMode(new DependencyObject()))
+                return default(ImageMoniker);
```

Comment:
Alternatively, we could just check if `imageService` is null and return `KnownMonikers.Document` as a fallback, even though it should never be null when running inside VS.

Reply:
What we return doesn't matter because these icons can't be shown in the designer anyway